### PR TITLE
Add payloads for waterway observation dataset

### PR DESCRIPTION
**Pull request by sacha-bydon**

### CODE REVIEW

File: jsonld-contexts/smbt-rol.jsonld
Comment (sacha-bydon) [inline jsonld-contexts/smbt-rol.jsonld:1]: Romain told me that I had to put the new attributes at the top, but it's okay if that's not the case

File: jsonld-contexts/waterway-observed-attributes.jsonld [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "@context": {
+    "ObservedWaterFlow": "https://smartdatamodels.org/ObservedWaterFlow",
+    "WeatherCondition": "https://smartdatamodels.org/WeatherCondition",
+    "adminComment": "https://smartdatamodels.org/adminComment",
+    "comment": "https://smartdatamodels.org/comment",
```
Comment (sacha-bydon) [inline jsonld-contexts/waterway-observed-attributes.jsonld]: `comment` attribute is already specified in `shellfish-farming.jsonld`

File: jsonld-contexts/smbt-rol.jsonld
Comment (bobeal) [reply jsonld-contexts/smbt-rol.jsonld:1]: If the context is clean (ie no redefinition of existing terms), the order does not matter. I hope this one is clean.

File: jsonld-contexts/waterway-observed-attributes.jsonld (hunk 1 above)
Comment (bobeal) [reply jsonld-contexts/waterway-observed-attributes.jsonld]: fixed.

**Review by bobeal — APPROVED** [review]

**Comment by sacha-bydon** [comment]
Everything look good to me

**Pull request merged**